Incorporate User Resources/Readme.md

```diff
@@ -1,59 +1,78 @@
+**Note:** You must save your component document before you can import files.
+
+### Referencing resource files in a WebVI
+You can reference a resource file using its relative URL. The relative URL is determined by the namespace of the WebVI and the namespace of the resource file.  
 
-User resources are referenced via a relative URL. This URL depends on what namespace the Web VI is in, and the namespace of the user resource.  
 Examples:
-- *Web VI:* Application:Main.gviweb  
-  *Resource:* Application:myImage.jpg  
-  *Relative URL:* myImage.jpg
-- *Web VI:* Application:ChildNamespace1:Main.gviweb  
-  *Resource:* Application:ChildNamespace2:myImage.jpg  
-  *Relative URL:* ../ChildNamespace2/myImage.jpg
-
-Images (*.png, *.bmp, *.jpg, *.jpeg, *.gif, *.svg, *.webp, *.ico):
-- From the front panel palette, expand the Drawings category, then drop a URL Image
-- Select the image control. In the right rail, set the Source URL property to the relative URL to the image you included.  
-Also, note that the URL should be URL-encoded. For example, to include "my image.jpg", the Source URL should be "my%20image.jpg".
-
-CSS stylesheets (*.css):
-- Go to the "HTML" editor of your Web VI. The active editor picker is directly below the document tab, near the top.
-- Find the `<head>` section.
-- Before the `</head>` line, add a reference to your CSS file, using its relative URL. For example:  
-`<link rel="stylesheet" type="text/css" href="main.css">`
-
-JavaScript files (*.js):
-- Go to the "HTML" editor of your Web VI. The active editor picker is directly below the document tab, near the top.
-- Find the `<head>` section.
-- Before the `</head>` line, add a reference to your JS file, using its relative URL. For example:  
-`<script src="myscript.js" type="text/javascript"></script>`
-
-HTML files (*.html, *.htm):
-- From the front panel palette, expand the Text category, then drop a Hyperlink Control.
-- Select the hyperlink. In the right rail, set the URL to the relative URL of the HTML file you included.
-
-# Usage
-This approach can be used to include existing resource files (such as images) and use them with a Web VI, without needing to upload those files to a web server ahead of time.
-
-The resource files can also be used to augment the Web VI's capabilities. Users can add other HTML widgets or scripts, and use them from the HTML editor view of the Web VI.
+- **WebVI:** `Application/Main.gviweb`
+  **Resource file:** `Application/myImage.jpg`
+  **Relative URL:** `myImage.jpg`
+- **WebVI:** `Application/ChildNamespace1/Main.gviweb`
+  **Resource:** `Application/ChildNamespace2/myImage.jpg`
+  **Relative URL:** `../ChildNamespace2/myImage.jpg`
+
+#### Referencing images (*.png, *.bmp, *.jpg, *.jpeg, *.gif, *.svg, *.webp, *.ico)
+1. On the panel of a WebVI, add a **URL Image** control.
+3. Select the URL image control. On the **Item** tab, set **Source URL** to the relative URL of the image you you want to reference.  
```

Comment:
2. not 3. It's rendered correctly in GitHub, but should probably still correct it in the source